racket/collects/racket/match/compiler.rkt

```diff
@@ -100,7 +100,7 @@
         #`[question (let ([tmps (accs #,x)] ...) body)])))
   (cond
     [(eq? 'box k)
-     (compile-con-pat (list #'unsafe-unbox*) #'box? (compose list Box-p))]
+     (compile-con-pat (list #'unsafe-unbox) #'box? (compose list Box-p))]
```

Comment:
I won't change this PR any further (besides for addressing PR feedback). But it looks like this `compose` should have been `compose1`?

Reply:
Oh, but it's at compile-time anyway. Wouldn't matter much.